### PR TITLE
findutils: 4.6.0 -> 4.7.0

### PR DESCRIPTION
**Pull request by lsix**
See https://lists.gnu.org/archive/html/info-gnu/2019-08/msg00009.html
for release information

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/tools/misc/findutils/default.nix [hunk 1]
```diff
@@ -3,20 +3,15 @@
 }:
 
 stdenv.mkDerivation rec {
-  name = "findutils-4.6.0";
+  name = "findutils-4.7.0";
```
Comment (lblasc) [inline pkgs/tools/misc/findutils/default.nix]: Switch to pname and version.
```
 stdenv.mkDerivation rec {
-  name = "findutils-4.6.0";
+  pname = "findutils";
+  version = "4.7.0";

   src = fetchurl {
-    url = "mirror://gnu/findutils/${name}.tar.gz";
-    sha256 = "178nn4dl7wbcw499czikirnkniwnx36argdnqgz4ik9i6zvwkm6y";
-  };
+    url = "mirror://gnu/findutils/${pname}-${version}.tar.xz";
+    sha256 = "16kqz9yz98dasmj70jwf5py7jk558w96w0vgp3zf9xsqk3gzpzn5";
+   };
```

**Comment by disassembler** [comment]
@GrahamcOfBorg build findutils

**Comment by lblasc** [comment]
Tested on x86_64, built rustc and ghc, haven't found any issue, also helps #66528

Thx!

**Comment by disassembler** [comment]
built successfully on linux.

**Pull request merged**